### ISSUE TITLE
[FEATURE] Version control.

### ISSUE THREAD

**Issue by AlexTryHarder** [jordan-dalby/ByteStash#26]
It would be nice to have version control, something in type that portainer has.
Editing code would add another version allowing to roll back in case of accidental override etc.
![image](https://github.com/user-attachments/assets/7eef7a5c-9ebe-489d-897f-de4cd3a72198)

**Comment by EDIflyer** [comment]
Good idea @AlexTryHarder - also I know there's a warning when deleting but I'd like the option of a trash folder for inadvertent deletions :)

**Comment by jordan-dalby** [comment]
> Good idea @AlexTryHarder - also I know there's a warning when deleting but I'd like the option of a trash folder for inadvertent deletions :)

Good idea, can you make a separate issue for that?

**Comment by EDIflyer** [comment]
No probs, @jordan-dalby - done!

**Comment by Falklian** [comment]
@jordan-dalby if this isn't being worked on, I wouldn't mind tackling it

**Comment by jordan-dalby** [comment]
> @jordan-dalby if this isn't being worked on, I wouldn't mind tackling it

Go ahead! I'll assign this ticket to you.
